### PR TITLE
[profiling] Dual-shipping notes.

### PR DESCRIPTION
**Pull request by AlexJF**
### What does this PR do?
Highlight caveats of profiling dual shipping, namely the priorities involving main and additional endpoints.

### Motivation
Clarifying operational details of profiling dual shipping.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/dual-shipping.md [hunk 1]
```diff
@@ -97,6 +97,9 @@ Requires Agent version >= 6.19 or 7.19.
 DD_APM_PROFILING_ADDITIONAL_ENDPOINTS='{\"https://intake.profile.datadoghq.com/api/v2/profile\": [\"apikey2\", \"apikey3\"], \"https://intake.profile.datadoghq.eu/api/v2/profile\": [\"apikey4\"]}'
 ```
 
+**Note:** Uploads to additional endpoints for the continuous profiler product are done through best-effort delivery.
+* The main endpoint will always have the highest priority. Uploads to additional endpoints are only handled after successful upload to the main endpoint.
+* Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints will simply be logged in the agent error logs.
```
Comment (gturbat) [inline content/en/agent/guide/dual-shipping.md:102]:
```suggestion
* Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints will be logged in the agent error logs.
```

**Review by gturbat — CHANGES_REQUESTED** [review]

File: content/en/agent/guide/dual-shipping.md [hunk 2]
```diff
@@ -97,6 +97,9 @@ Requires Agent version >= 6.19 or 7.19.
 DD_APM_PROFILING_ADDITIONAL_ENDPOINTS='{\"https://intake.profile.datadoghq.com/api/v2/profile\": [\"apikey2\", \"apikey3\"], \"https://intake.profile.datadoghq.eu/api/v2/profile\": [\"apikey4\"]}'
 ```
 
+**Note:** Uploads to additional endpoints for the Continuous Profiler product are done through best-effort delivery.
+* The main endpoint has the highest priority. Uploads to additional endpoints are only handled after successful upload to the main endpoint.
+* Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints will be logged in the agent error logs.
```
Comment (gturbat) [inline content/en/agent/guide/dual-shipping.md:102]:
```suggestion
* Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints are logged in the agent error logs.
```

**Review by gturbat — APPROVED** [review]

File: content/en/agent/guide/dual-shipping.md [hunk 3]
```diff
@@ -97,6 +97,9 @@ Requires Agent version >= 6.19 or 7.19.
 DD_APM_PROFILING_ADDITIONAL_ENDPOINTS='{\"https://intake.profile.datadoghq.com/api/v2/profile\": [\"apikey2\", \"apikey3\"], \"https://intake.profile.datadoghq.eu/api/v2/profile\": [\"apikey4\"]}'
 ```
 
+**Note:** Uploads to additional endpoints for the Continuous Profiler product are done through best-effort delivery.
+* The main endpoint has the highest priority. Uploads to additional endpoints are only handled after successful upload to the main endpoint.
+* Responses from additional endpoints are not forwarded back to the profiler. Any errors during delivery to additional endpoints are logged in the agent error logs.
```
Comment (hestonhoffman) [inline content/en/agent/guide/dual-shipping.md:102]:
```suggestion
**Note:** Uploads to additional endpoints for the Continuous Profiler product are done through best-effort delivery.
* The main endpoint has the highest priority. Uploads to additional endpoints are only handled after uploads to the main endpoint have completed successfully.
* Responses from additional endpoints are not forwarded back to the profiler. Any errors during delivery to additional endpoints are logged in the Agent error logs.
```

**Review by hestonhoffman — COMMENTED** [review]
Some small edits

**Pull request merged**